src/execution_plan/ops/op.c

```
@@ -25,6 +25,16 @@ OpResult _OpBase_reset_noop
 	return OP_OK;
 }
 
+Record _OpBase_consume
+(
+	OpBase *op
+) {
+	ASSERT(op != NULL);
```

cppcheck (2.10-2)

[error] Null pointer dereference

(nullPointer)